CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Added `mremap` (#[1306](https://github.com/nix-rust/nix/pull/1306))
 
+- Added `personality` (#[1331](https://github.com/nix-rust/nix/pull/1331))
+### Changed
```

Comment:
As in the other PR, `### Changed` got duped.

Reply:
Whoops. Fixed here and there.